docs/docs/userguide/CONFIGURATION.md

```diff
@@ -107,12 +107,25 @@ Final domain will have the following format: `<product>.<environment-name>.<doma
 domain="<domain-name>" # for example: "mydomain.com"
 ```
```

Comment:
I think better we move the line 104 after this code block (first we suggest the format and then we refer to the example we provided in line 107 to show the final result)

Reply:
Done